yarn.lock

```diff
@@ -1551,10 +1551,10 @@
   dependencies:
     "@babel/types" "^7.20.7"
 
-"@types/backbone@^1.4.15":
-  version "1.4.19"
-  resolved "https://registry.npmjs.org/@types/backbone/-/backbone-1.4.19.tgz#f6e8406fed40ca3fe224f6e59115142b62be76d6"
-  integrity sha512-byyn236JymGByOajKA7mi1k+/jKn162TIvArOB4SHgOGbVlFj8CSfJH4jekP0qo0vJwW5khrrsiiO1Jsos6ZvA==
+"@types/backbone@1.4.15":
```

Comment:
@artf Seems need to fix to 1.4.19, because 1.4.15 cause error with underscore typings

Reply:
hi @Delagen can you provide a replication of your issue?

Reply:
I have this error
```
TS2694: Namespace '"D:/.../node_modules/@types/und
erscore/index"' has no exported member 'ListIterator'.
```

Reply:
May be is caused by "module": "ESNext","useDefineForClassFields": true,